src/pug/blog/report_17.pug

```diff
@@ -0,0 +1,53 @@
+extends _blog.pug
+
+block append subtitle
+  | Progress report #17
+
+block content
+  h2 Progress report #17
+  +news_header('6 October 2023', 'report_16', '')
+
+  h3 Ctrl app
+  p We released firmware #[+ax(org.fw_release+'0.92.0') 0.92.0] along the new #[+ax('https://ctrl.inputlabs.io') Ctrl app] for the Alpakka controller. We decided that having a minimalistic app was the best solution to address the feedback from the community, and the desire to have a simpler way to modify profiles. The app is aligned with Input Labs principles, so it is completely #[em optional], #[em open-source], and #[em multi-platform].
+  p Some of the features the Ctrl app have already:
+  ul
+    li Changing controller configuration (mouse sensitivity, touch responsiveness, deadzone, protocol).
+    li Functions such as calibration and factory reset.
+    li Simple access to the firmware logs, for troubleshooting any issues while DIY'ing the controller.
+
+  p Features planned for later:
+  ul
+    li Profile edit.
+    li Offline mode (based on PWA).
+    li Input tester.
+
+  p The app is accessible at #[+ax('https://ctrl.inputlabs.io') ctrl.inputlabs.io] and it is compatible with #[em Windows], #[em Linux], and the #[em Steam Deck].
```

Comment:
You may want to write that it requires a browser that supports webusb, i.e. presently Chrome, Chromium, Edge, and Safari

Reply:
Mac OS